sftp.c

```diff
@@ -1778,6 +1797,20 @@ parse_dispatch_command(struct sftp_conn *conn, const char *cmd, char **pwd,
 			err = 1;
 		}
 		break;
+	case I_CLR_RCPT:
+		iron_reset_recipients();
+		break;
+	case I_SHOW_RCPT:
+		iron_list_recipients();
```

Comment:
Naming nitpick: I find it weird that the command is `showrcpt` and the function is `iron_list_recipients`. Feels like the function should be `iron_show_recipients` or else the command should be `lsrcpt` or something.